README.md

```diff
@@ -13,37 +13,147 @@ Our focus is to explore a variety of software applications to ensure we have goo
 This example has been used by at least one internal or external development team to create a series of jobs that successfully rendered. However, your mileage may vary. If you have questions or issues with this example, please start a discussion or cut an issue.
 ---
 
-AWS Deadline Cloud for Cinema 4D is a python package that allows users to create [AWS Deadline Cloud][deadline-cloud] jobs from within Cinema 4D. Using the [Open Job Description (OpenJD) Adaptor Runtime][openjd-adaptor-runtime] this package also provides a command line application that adapts Cinema 4D's command line interface to support the [OpenJD specification][openjd].
+AWS Deadline Cloud for Cinema 4D is a python package that allows users to create [AWS Deadline Cloud][deadline-cloud] jobs from within Cinema 4D. It provides both the implementation of a Cinema 4D extension for your workstation that helps you offload the computation for your rendering workloads
+to [AWS Deadline Cloud][deadline-cloud] to free up your workstation's compute for other tasks, and the implementation of a command-line
+adaptor application based on the [Open Job Description (OpenJD) Adaptor Runtime][openjd-adaptor-runtime] that improves AWS Deadline Cloud's
+ability to run Cinema 4D efficiently on your render farm.
+
 
 [deadline-cloud]: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/what-is-deadline-cloud.html
 [deadline-cloud-client]: https://github.com/aws-deadline/deadline-cloud
 [openjd]: https://github.com/OpenJobDescription/openjd-specifications/wiki
 [openjd-adaptor-runtime]: https://github.com/OpenJobDescription/openjd-adaptor-runtime-for-python
 [openjd-adaptor-runtime-lifecycle]: https://github.com/OpenJobDescription/openjd-adaptor-runtime-for-python/blob/release/README.md#adaptor-lifecycle
-
+[service-managed-fleets]: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/smf-manage.html
+[default-queue-environment]: https://docs.aws.amazon.com/deadline-cloud/latest/userguide/create-queue-environment.html#conda-queue-environment
 
 ## Compatibility
 
 This library requires:
 
-1. Cinema 4D 2023 - 2024,
-1. Python 3.9 or higher; and
-1. Linux, Windows, or a macOS operating system.
+1. Cinema 4D 2024 - 2025
+1. Python 3.9 or higher; but Python 3.11 is recommended as this is the version Cinema 4D uses natively.
+1. Windows
```

Comment:
There are some instructions for Linux and Mac. Are they partially working but not supported? Might be helpful to clarify that here.

Reply:
> Are they partially working but not supported?

We can support Mac submitter but Mac does not have an official commandline rendering application for Mac so we don't support it for the adaptors for Cinema 4D. 

Linux cannot be installed as submitters as only commandling rendering is available but it has not been fully tested. 

I'll update it here.